contracts/verifiers/UniversalVerifier.sol

```diff
@@ -92,9 +92,7 @@ contract UniversalVerifier is
         uint64[] calldata requestIds,
         ZKPRequest[] calldata requests
     ) public override(RequestOwnership, ValidatorWhitelist, ZKPVerifierBase) {
-        for (uint256 i = 0; i < requestIds.length; i++) {
-            setZKPRequest(requestIds[i], requests[i]);
```

Comment:
We need to call `setZKPRequest` in UniversalVerifier instead of `super.setZKPRequest` to emit the event `ZKPRequestSet` for these requests.
So this can't be optimized like this.